seeds/3_files.js

```diff
@@ -3,46 +3,48 @@
 var fs = require('fs');
 
 exports.seed = function(knex, Promise) {
+  // As knex does not have an .insertIfNotExists(), we hook into catch()
+  // so that seeding doesn't break when running on an already seeded database.
   return Promise.join(
     knex('files').insert({
       project_id: 1,
       path: '/spacecats-API/index.js',
       buffer: fs.readFileSync(__dirname + '/file_data/index.js')
-    }),
+    }).catch(function() { return knex('files'); }),
```

Comment:
this has been removed again based on the conclusion that the seeds are for testing only, so will always operate on an empty db, where there is no risk over appending to preexisting data (this PR builds on top of https://github.com/mozilla/publish.webmaker.org/pull/171 which got updated to reflect that)